### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by LethaboMot**
In this project, we Added

-we edited the menu items
-Added the transition with the CSS 
-Added Javascript

### CODE REVIEW

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @LethaboMot and @eric2umeh, 

# Required Changes ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✅ 
- Descriptive PR ☑️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Please 🙏🏽 replace all the sections with the corresponding information of your project, the title should be replaced with the title of your project 👇🏽 

image

- [ ] Kindly think of a common user that doesn't have knowledge of programmatic, ¿How could you help them to use your repository?, please 🙏🏽 complete this section with the instruction to help a common user.

image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,343 +1,272 @@
...
-
-        <div class="info-tag">
-          <img
-            src="images/Hamburger button.svg"
-            alt="three lines"
-            width="24"
-            height="24"
-          />
+        <div id="slidemenu" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><img src="./images/xicon.svg" alt="x icon" /></a>
```
Comment (Ismailco) [inline index.html:18]: - [ ] As the previous reviewer mentioned, kindly make sure to fix the close button for the mobile menu. 
**hint** I can't see a `xicon.svg` image in the images folder

Screenshot_20221220_174915

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @LethaboMot, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi Team👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️
* I like the menu's animation 💯  


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**